main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	olmscheme "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned/scheme"
```

Comment:
Do we specifically need the `olmscheme` here? If there is no specific reason, we can just use the k8s scheme `"k8s.io/client-go/kubernetes/scheme"`.